README.md

```diff
@@ -1,19 +1,24 @@
-# ACGPs - AragonCoop Governance Proposals
+# ACGPs - AragonCooperative[DAO] Governance Proposals
 
-This is a space where AragonCoop Community members can create and submit Governance Proposals to be executed/obeyed by the AragonCoop members and its delegates (if any).
+This is a space where AragonCooperativeDAO community members can create and submit Governance Proposals to be enacted by AragonCooperativeDAO members and delegates (if any).
 
-As we are starting a governance framework practically from zero, we propose to start operating our Governance over the rinkeby network. Once the Governance framework has shown itself to be stable enough, a migration to the mainnet shall be proposed and executed.
+As we are exploring a governance framework from near zero, we propose to initially operate our governance over the rinkeby network (also exploring experimental DAO tools). Once a bare-bones cooperative governance framework and functional toolset has demonstrated itself reasonably stable, migration to mainnet shall be proposed.
 
-## Where are we taking decisions?
-There are two organizations where the [AragonCoop Members](https://github.com/aragoncoop/membership) have their membership Tokens.
+## Decision Locations?
+There are two on-chain locations where [AragonCooperativeDAO Members](https://github.com/aragoncoop/membership) host their two types of AragonCooperativeDAO tokens: MBR (1M1V - aka the 'Golden Ticket' - only ever one NFT token per member) &: MRT (rep-based membership token).
```

Comment:
Even though the token is non-transferrable and limited to one per address, I would not use NFT here as it refers to a specific token standard.